bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "unicorn-engine"
+version.workspace = true
+authors.workspace = true
+keywords.workspace = true
+categories.workspace = true
+documentation.workspace = true
+edition.workspace = true
+license.workspace = true
+readme.workspace = true
+repository.workspace = true
+description.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+unicorn-engine-sys = { version = "2.1.3", path = "sys", features = [
+], default-features = false }
```

Comment:
nit: why break into two lines?

Reply:
[taplo](https://github.com/tamasfe/taplo) did that - I'll make it 1 line